corehq/apps/app_manager/views/view_generic.py

```diff
@@ -342,6 +342,9 @@ def _make_name(suffix):
     confirm = request.session.pop('CONFIRM', False)
     context.update({'confirm': confirm})
 
+    release_control_visible = AppReleaseModeSetting.get_settings(domain)
+    context.update({'show_release_mode': release_control_visible.is_visible})
```

Comment:
nit: this variable `release_control_visible` seems redundant here.

Reply:
It is used here - https://github.com/dimagi/commcare-hq/pull/32245/files#diff-cf519579ec6065016857e65ebdc75d47106708c1bb2022dbb47f14d927395b8aR44